src/main/java/com/redhat/devtools/intellij/quarkus/run/QuarkusOpenAppInBrowserAction.java

```diff
@@ -25,6 +25,9 @@ public QuarkusOpenAppInBrowserAction() {
     @Override
     public void actionPerformed(@NotNull AnActionEvent e) {
         TelemetryService.instance().action(TelemetryService.UI_PREFIX + "openApplication").send();
-        BrowserUtil.browse(QuarkusRunContext.getContext(e).getApplicationURL(), PlatformDataKeys.PROJECT.getData(e.getDataContext()));
+        QuarkusRunContext context = QuarkusRunContext.getContext(e);
```

Comment:
This is not required because action should not be visible if context is null (see parent class)